src/store/reducers/pdisk/utils.ts

```diff
@@ -60,11 +63,19 @@ export function preparePDiskDataResponse([pdiskResponse = {}, nodeResponse]: [
     preparedVDisks.sort((disk1, disk2) => Number(disk2.VDiskSlotId) - Number(disk1.VDiskSlotId));
 
     const vdisksSlots: SlotItem<'vDisk'>[] = preparedVDisks.map((preparedVDisk) => {
+        // VDisks do not have strict limits and can be bigger than they should
+        // In most storage views we don't colorize them depending on space usage
+        // Colorize them inside PDiskSpaceDistribution so overused slots will be visible
+        const slotSeverity = Math.max(
+            getSpaceSeverity(preparedVDisk.AllocatedPercent),
+            preparedVDisk.Severity || 0,
+        );
```

Comment:
We don't use allocated percent to get vdiks severity inside prepare functions, we use it only here while preparing slots